build.sbt

```diff
@@ -1,17 +1,17 @@
 
-name := "spark-llap"
-version := "1.0"
+name := "spark-llap_2.11"
+version := "2.0"
 organization := "com.hortonworks"
-scalaVersion := "2.10.5"
-val scalatestVersion = "2.2.4"
+scalaVersion := "2.11.8"
+val scalatestVersion = "2.2.6"
 
-sparkVersion := sys.props.getOrElse("spark.version", "1.6.0")
+sparkVersion := sys.props.getOrElse("spark.version", "2.0.0")
 
 
-val hadoopVersion = sys.props.getOrElse("hadoop.version", "2.7.1")
-val hiveVersion = sys.props.getOrElse("hive.version", "2.1.0-SNAPSHOT")
+val hadoopVersion = sys.props.getOrElse("hadoop.version", "2.7.3")
+val hiveVersion = sys.props.getOrElse("hive.version", "2.1.0.2.5.0.0-1245")
```

Comment:
Ur, IMHO, `2.1.0.2.5.0.0-1245` is better since `2.1.0-SNAPSHOT` seems to change time to time. If you're okay, let's use this as a default value.